main.pm

```diff
@@ -92,6 +92,10 @@ sub cleanup_needles() {
 
 }
 
+sub is_staging () {
+    return get_var('STAGING');
```

Comment:
Yeah but, then we'll be different between openSUSE (where staging is a Flavour) and SLE (where staging is a Version) and I'd rather avoid that..